logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -63,6 +68,24 @@ class ConversationDataSource(
         return conversationDAO.getAllMembers(idMapper.toDaoModel(conversationId)).first().map { idMapper.fromDaoModel(it.user) }
     }
 
+    override suspend fun persistMember(member: Member, conversationID: QualifiedID): Either<CoreFailure, Unit> {
```

Comment:
IMO `addMember` / `deleteMember` makes more sense, that's gonna get persisted is implied since it's a DAO.

Reply:
Agree, I thought of it. but I wanted to keep the consistency [the rest dao classes use persist - we need to agree on this naming conventions]